corehq/middleware.py

```diff
@@ -156,7 +157,14 @@ def process_response(self, request, response):
             response['Expires'] = "-1"
             response['Pragma'] = "no-cache"
         else:
+            content_type, _ = mimetypes.guess_type(request.path)
             response['Cache-Control'] = "max-age=31536000"
+            del response['Vary']
```

Comment:
Hmmm, nice! (Actually I wish `del dict[key]` worked like that, although I guess that would break the "Errors should never pass silently" thing.)